### PR TITLE
Integrate vi and i2ctool into builtin apps for tash

### PR DESCRIPTION
**Pull request by tadziopazur**
These tools come from apps/system.

### CODE REVIEW

File: apps/system/i2c/Makefile [hunk 1]
```diff
@@ -125,8 +127,17 @@ install:
 
 endif
 
+ifeq ($(CONFIG_SYSTEM_I2CTOOL),y)
```
Comment (jeongchanKim) [inline apps/system/i2c/Makefile]: This is only when CONFIG_BUILTIN_APPS is enabled.
let's change to ifeq ($(CONFIG_BUILTIN_APPS),$(CONFIG_SYSTEM_I2CTOOL),yy)

File: apps/system/vi/Makefile [hunk 2]
```diff
@@ -128,8 +132,17 @@ install:
 
 endif
 
+ifeq ($(CONFIG_SYSTEM_VI),y)
```
Comment (jeongchanKim) [inline apps/system/vi/Makefile]: same as above

**Comment by sunghan-chang** [comment]
@tadziopazur Could you let us know why do you want to use it at TASH? VI is an imcomplete applicaion.

**Comment by tadziopazur** [comment]
Vi might be incomplete, but is useful for modifying short javascript files.

File: apps/system/i2c/Makefile [hunk 3]
```diff
@@ -125,8 +127,17 @@ install:
 
 endif
 
+ifeq ($CONFIG_BUILTIN_APPS)$(CONFIG_SYSTEM_I2CTOOL),yy)
```
Comment (rzr) [inline apps/system/i2c/Makefile]: mistake fixed in:
https://github.com/Samsung/TizenRT/pull/2439#event-1959436602

**Pull request merged**